### PR TITLE
Just have accept-pr be a noop, which will naturally not succeed if de…

### PR DESCRIPTION
**Pull request by anuraaga**
…pendencies fail.

Had `accept-pr` fail a perfectly good build on me. While I'm not sure, isn't it enough to have a no-op job instead of using the workflow-conclusion action?

### CODE REVIEW

**Review by trask — COMMENTED** [review]
I think this is what @iNikem originally tried (see #1498), but IIRC the github branch protection rule doesn't prevent merging on missing job, only on failed job.

**Review by trask — APPROVED** [review]
approving to see if merge button lights up even though `accept-pr` job was skipped

**Comment by anuraaga** [comment]
Oh well - hopefully the accept-pr doesn't flake too much :)

**Pull request closed without merging**
